Wrapping/Generators/Python/PyUtils/itkPyCommand.cxx

```diff
@@ -39,6 +39,7 @@ namespace itk
 PyCommand::PyCommand()
 {
   this->m_Object = nullptr;
+  this->m_EmptyArgumentList = Py_BuildValue("", 0);
```

Comment:
We may need an empty tuple here. [Py_BuildValue](https://docs.python.org/3/c-api/arg.html#c.Py_BuildValue) does not always build a tuple.

Reply:
Thank you. This lead me to the solution.